lib/commands/env.js

```diff
@@ -351,4 +351,80 @@ env._printEnv = function (env) {
   xervo.io.print(env.name.yellow + ' = ' + env.value.grey)
 }
 
+// Find the index for the specified env var in the specified env var array.
+//    Returns -1 if not found.
+//
+var envVarInArray = function (envVars, name) {
+  for (var idx = 0; idx < envVars.length; idx++) {
+    if (envVars[idx].name.toUpperCase() === name.toUpperCase()) return idx
+  }
+
+  return -1
+}
+
+// ------------------------------------------------------------------------------
+/**
+ * Displays a prompt for choosing an envVar from a list.
+ * @param {Array} envVars Array of a project envVars.
+ * @param {String} [envVar] Default selection.
+ * @param {Function} cb
+ */
+// ------------------------------------------------------------------------------
+env.chooseEnvVarPrompt = function (envVars, envVar, cb) {
+  var envVarIndex = -1
+
+  if (typeof envVar === 'function') {
+    cb = envVar
+    envVar = null
+  }
+  // A default selection has been made. See if a match exists in the collection.
+  if (typeof envVar === 'string' && envVar !== '') {
+    envVarIndex = envVarInArray(envVars, envVar)
```

Comment:
yeah. I think we can bump up the required version soon, but we need to give some heads up about it first